Core/src/org/sleuthkit/autopsy/casemodule/Bundle.properties

```diff
@@ -223,17 +223,11 @@ ReviewModeCasePanel.MetadataFileHeaderText=Metadata File
 OpenMultiUserCasePanel.jLabel1.text=Recent Cases
 OpenMultiUserCasePanel.openButton.text=Open
 OpenMultiUserCasePanel.cancelButton.text=Cancel
-MultiUserCasesPanel.rbWeeks.text=Weeks
-MultiUserCasesPanel.rbDays.text=Days
-MultiUserCasesPanel.bnShowLog.toolTipText=Display case log file for selected case
-MultiUserCasesPanel.bnShowLog.text=&Show Auto Ingest Case Log
-MultiUserCasesPanel.rbAllCases.text=Everything
-MultiUserCasesPanel.bnRefresh.text=&Refresh
 MultiUserCasesPanel.bnOpen.text=&Open
-MultiUserCasesPanel.rbGroupLabel.text=Show cases created in the last 10:
-MultiUserCasesPanel.rbMonths.text=Months
 CueBannerPanel.newCaseLabel.text=New Case
 CueBannerPanel.openCaseButton.text=
 CueBannerPanel.openCaseLabel.text=Open Case
 MultiUserCasesPanel.bnOpenSingleUserCase.text=Open Single-User Case...
 CueBannerPanel.newCaseButton.text=
+MultiUserCasesPanel.searchLabel.text=Start typing to search by case name
```

Comment:
Just an initial thought as I run this: It says "Start typing to search by case name", but this doesn't work unless the user first selects a row in the table. It's a little misleading. Perhaps this should be changed to either only show that message when a row has been selected, or figure out a way to get the search working without making that initial selection.

Reply:
I will make it auto select the first element unless you see an issue with that idea.  Although they would still be able to prevent it from working by un-selecting the node with ctrl+click.  
Alternatively I would prefer to just make the message longer than to make the message changing based on selection.

Reply:
I am fine with adding to the help text.